README.md

```diff
@@ -169,17 +171,17 @@ The `createClient` method takes a WebDAV service URL, and a configuration option
-| `username`    | _None_        | Username for authentication.                      |
-| `withCredentials` | _None_    | Credentials inclusion setting for the request,    |
+| Option            | Default                                                                                   | Description                                                                                                                               |
+| ----------------- | ----------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| `authType`        | `null`                                                                                    | The authentication type to use. If not provided, defaults to trying to detect based upon whether `username` and `password` were provided. |
+| `contactHref`     | _[This URL](https://github.com/perry-mitchell/webdav-client/blob/master/LOCK_CONTACT.md)_ | Contact URL used for LOCKs.                                                                                                               |
+| `headers`         | `{}`                                                                                      | Additional headers provided to all requests. Headers provided here are overridden by method-specific headers, including `Authorization`.  |
+| `httpAgent`       | _None_                                                                                    | HTTP agent instance. Available only in Node. See [http.Agent](https://nodejs.org/api/http.html#http_class_http_agent).                    |
+| `httpsAgent`      | _None_                                                                                    | HTTPS agent instance. Available only in Node. See [https.Agent](https://nodejs.org/api/https.html#https_class_https_agent).               |
+| `password`        | _None_                                                                                    | Password for authentication.                                                                                                              |
+| `token`           | _None_                                                                                    | Token object for authentication.                                                                                                          |
+| `username`        | _None_                                                                                    | Username for authentication.                                                                                                              |
+| `withCredentials` | _None_                                                                                    | Credentials inclusion setting for the request,                                                                                            |
```

Comment:
This is actually harder to read in a width-constricted terminal, when compared to the shortened version that was there before. Are you using some markdown formatting utility perhaps?